binary_transparency/firmware/internal/verify/bundle.go

```diff
@@ -22,12 +22,13 @@ import (
 	"fmt"
 
 	"github.com/google/trillian-examples/binary_transparency/firmware/api"
+	"github.com/google/trillian-examples/binary_transparency/firmware/internal/crypto"
 )
 
 // BundleForUpdate checks that the manifest, checkpoint, and proofs in a bundle
 // are all self-consistent, and that the provided firmware image hash matches
 // the one in the bundle.
-func BundleForUpdate(b api.ProofBundle, fwHash []byte) error {
+func BundleForUpdate(b api.ProofBundle, fwHash []byte, dc api.LogCheckpoint, proof [][]byte) error {
```

Comment:
proof -> cp (or cProof)

Reply:
Renamed it to cProof for more clarity here!